src/test/java/api/loans/DueDateNotRealTimeScheduledNoticesProcessingTests.java

```diff
@@ -396,20 +401,23 @@ void missingReferencedEntitiesDoNotBlockProcessing() {
     ItemResource times = itemsFixture.basedUponInterestingTimes();
     ItemResource uprooted = itemsFixture.basedUponUprooted();
     ItemResource dunkirk = itemsFixture.basedUponDunkirk();
+    ItemResource lotr = itemsFixture.basedUponLotr();
 
     // loans
     IndividualResource nodToJames = checkOutFixture.checkOutByBarcode(nod, james, loanDate.plusHours(1));
     IndividualResource temeraireToJames = checkOutFixture.checkOutByBarcode(temeraire, james, loanDate.plusHours(2));
-    IndividualResource planetToJames = checkOutFixture.checkOutByBarcode(planet, james, loanDate.plusHours(3));
-    checkOutFixture.checkOutByBarcode(times, steve, loanDate.plusHours(4));
-    IndividualResource uprootedToSteve = checkOutFixture.checkOutByBarcode(uprooted, steve, loanDate.plusHours(5));
-    checkOutFixture.checkOutByBarcode(dunkirk, jessica, loanDate.plusHours(6));
+    checkOutFixture.checkOutByBarcode(lotr, charlotte, loanDate.plusHours(3));
+    IndividualResource planetToJames = checkOutFixture.checkOutByBarcode(planet, james, loanDate.plusHours(4));
+    checkOutFixture.checkOutByBarcode(times, steve, loanDate.plusHours(5));
+    IndividualResource uprootedToSteve = checkOutFixture.checkOutByBarcode(uprooted, steve, loanDate.plusHours(6));
+    checkOutFixture.checkOutByBarcode(dunkirk, jessica, loanDate.plusHours(7));
 
+    usersClient.replace(charlotte.getId(), new UserBuilder().withPatronGroupId(null));
```

Comment:
The test's name is `missingReferencedEntities`, and here you are deleting the reference, not the entity

Reply:
New test with appropriate name has been created